lib/handler/mimemap/defaults.c.h

```diff
@@ -41,6 +41,7 @@ MIMEMAP("jpeg", "image/jpeg")
 MIMEMAP("jpg", "image/jpeg")
 MIMEMAP("jxr", "image/jxr")
 MIMEMAP("js", "application/javascript")
+MIMEMAP("mjs", "applicaftion/javascript")
```

Comment:
Hello @jxck , thank you for opening the PR. It looks like there's a typo here in `application`